src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -0,0 +1,70 @@
+package lotto.domain;
+
+import java.util.Objects;
+
+public class LottoNumber implements Comparable<LottoNumber> {
+
+    private static final int MIN_NUMBER_OF_LOTTO = 1;
+    private static final int MAX_NUMBER_OF_LOTTO = 45;
+    private final int number;
```

Comment:
```suggestion
    private static final int MAX_NUMBER_OF_LOTTO = 45;
    
    private final int number;
```

소소하지만 사이 공백을 추가한다면 가독성이 상승합니다 :)